4-tools/sieve.py

```diff
@@ -22,4 +22,4 @@ def sieve(n):
 
 
 primes = sieve(5000)
-print(len(primes))
+print(len(primes))  # noqa: T201
```

Comment:
Why isn't this guarded by `if __name__ == "__main__":`?

Reply:
The function is not meant to be run as such (it can't be because `@profile` is not valid syntax), but adding `if __name__ == "__main__":` doesn't hurt and makes it look better, so I will add it.